charts/prometheus-logstash-exporter/templates/deployment.yaml

```diff
@@ -0,0 +1,147 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ template "logstash-exporter.fullname" . }}
+  labels:
+    chart: {{ template "logstash-exporter.chart" . }}
```

Comment:
please use the more recent labeling defaults which are using templating.

you can copy it from the chart created by:

```
helm create prometheus-logstash-exporter
```